tensorflow_datasets/core/visualization/audio_visualizer.py

```diff
@@ -0,0 +1,68 @@
+""" Audio Visualization
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+
+import random
+import IPython.display
```

Comment:
This will crash for all user not running in a non-ipython environement. This should not be imported in the global scope